### PR TITLE
Change `RSpec/ContextWording` cop to always report an offense when both `Prefixes` and `AllowedPatterns` are empty

### PR DESCRIPTION
**Pull request by ydah**
see: https://github.com/rubocop/rubocop-rspec/pull/1972#discussion_r1808522184

______________________________________________________________________

Before submitting the PR make sure the following are checked:

- [x] Feature branch is up-to-date with `master` (if not - rebase it).
- [x] Squashed related commits together.
- [x] Added tests.
- [x] Updated documentation.
- [x] Added an entry to the `CHANGELOG.md` if the new code introduces user-observable changes.
- [x] The build (`bundle exec rake`) passes (be sure to run this locally, since it may produce updated documentation that you will need to commit).

### CODE REVIEW

File: lib/rubocop/cop/rspec/context_wording.rb [hunk 1]
```diff
@@ -58,7 +61,9 @@ module RSpec
       class ContextWording < Base
         include AllowedPattern
 
-        MSG = 'Context description should match %<patterns>s.'
+        MSG_MATCH = 'Context description should match %<patterns>s.'
+        MSG_ALWAYS = 'Current settings are always to be offense. Please add ' \
```
Comment (bquorning) [inline lib/rubocop/cop/rspec/context_wording.rb:65]:
```suggestion
        MSG_ALWAYS = 'Current settings will always report an offense. Please add ' \
```

File: spec/rubocop/cop/rspec/context_wording_spec.rb [hunk 2]
```diff
@@ -220,9 +220,10 @@
       { 'Prefixes' => [], 'AllowedPatterns' => [] }
     end
 
-    it 'skips any description' do
-      expect_no_offenses(<<~RUBY)
-        context 'arbitrary text' do
+    it 'always occurs an offense' do
```
Comment (bquorning) [inline spec/rubocop/cop/rspec/context_wording_spec.rb:223]:
```suggestion
    it 'always registers an offense' do
```

File: lib/rubocop/cop/rspec/context_wording.rb [hunk 3]
```diff
@@ -12,6 +12,9 @@ module RSpec
       #
       # @see http://www.betterspecs.org/#contexts
       #
+      # If both `Prefixes` and `AllowedPatterns` are empty, this cop will always
+      # be offense. So you need to set at least one of them.
```
Comment (bquorning) [inline lib/rubocop/cop/rspec/context_wording.rb:16]:
```suggestion
      # report an offense. So you need to set at least one of them.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,8 @@
 
 ## Master (Unreleased)
 
+- Change `RSpec/ContextWording` cop to always occur offenses when both `Prefixes` and `AllowedPatterns` are empty. ([@ydah])
```
Comment (bquorning) [inline CHANGELOG.md:5]:
```suggestion
- Change `RSpec/ContextWording` cop to always report an offense when both `Prefixes` and `AllowedPatterns` are empty. ([@ydah])
```

**Review by bquorning — COMMENTED** [review]
This PR is exactly what I had in mind 😄

I am not a native English speaker, but we usually use “report an offense”, so I made a few such suggestions.

**Comment by ydah** [comment]
Thank you for your review! I updated this PR.

**Review by bquorning — APPROVED** [review]

**Pull request merged**